src/main/java/com/sprint/mission/discodeit/controller/AuthController.java

```diff
@@ -22,13 +27,26 @@ public class AuthController implements AuthApi {
 
   private final AuthService authService;
 
-  @PostMapping(path = "login")
-  public ResponseEntity<UserDto> login(@RequestBody @Valid LoginRequest loginRequest) {
-    log.info("로그인 요청: username={}", loginRequest.username());
-    UserDto user = authService.login(loginRequest);
-    log.debug("로그인 응답: {}", user);
+
+  @GetMapping("/csrf-token")
+  public ResponseEntity<CsrfToken> getCsrfToken(CsrfToken csrfToken) {
+    log.debug("CSRF 토큰 요청");
+    return ResponseEntity.status(HttpStatus.OK).body(csrfToken);
+  }
+
+  @GetMapping("/me")
+  public ResponseEntity<UserDto> getCurrentUser() {
+    UserDto currentUser = authService.getCurrentUser();
+    return ResponseEntity.ok(currentUser);
+  }
```

Comment:
```suggestion
  @GetMapping("/me")
  public ResponseEntity<UserDto> getCurrentUser(@AuthenticationPrincipal UserDetails userDetails) {
    UserDto currentUser = authService.getCurrentUser();
    return ResponseEntity.ok(currentUser);
  }
```

이렇게 하면 UserDetails를 컨트롤러에서 받을 수 있습니다. 서비스에 static으로 접근지 않아도 돼요~~